### DISCUSSION TITLE
Lower new created arena team MMR

### DISCUSSION THREAD

**Discussion by Reily93** [chromiecraft/chromiecraft#7359] [Suggestions]
Hi creating a new team give u 1500 starting mmr (u dont see it) and it get lowered or hightered to match ur elo, this works on high populated servers, what happen atm is that making a new team is a frustrating experience for the new players , which they found to be prey of high team cause they get more points then a highter team <=1500 . This way making a new team is good way to feed rating to  higher teams.
 Wintrading is not only one team stay immobile when the other one get wthe win is also when , 1 team lower in skill join costantly into a highter team ( presumibly same guild or not necessarily) with the knowledge they cant win and still get a lose streak highter than 10 games to help the hightere team to get his rating

so a good mmr to start a new team could be a flat 1000 or lower , this way lower team get matched istantly and not incouring into highter ones , without the fear someone gonna join into them

**Comment by Helias** [comment]
Hi, we are considering it seriously, we will let you know when we'll take any action about this.
Hope that after the change you'll be ready to give us a good feedback, because estimating the right starting MMR for balancing the points is not easy unfortunately

**Reply by Helias** [reply]
I set `750`, we'll experiment with this, please @Reily93 report any feedback about it

**Comment by Reily93** [comment]
Ciao Helias , sorry i didnt saw u answered so i apologize, btw yeah u dont know which effects gonna have on arena but  the only way to find out is test it for a short time to see how it goes , otherwise we put it back, cause when honey fixed the queue  waiting time , we had fast aena wich is good but  was a dobule sword edge sword cause u can farm low rating teams, so lets see if this strat gonna balance the previous one, thx btw for reaching it!

**Comment by Reily93** [comment]
we just did 1 test , we made 1 team with one of my alt that never played arena cause fresh 70, i checked the site and i start with 1500, so its not working somehow, probably also personal mmr for the first arena has to be setted down

**Reply by Helias** [reply]
we need to reload conf or a restart, when the server will restart, it'll be 750, I forgot to say

**Comment by Reily93** [comment]
hi i think this measure is counterproductive ,  the difficulty to raise the rating became a serious issue and after all that time testing it, would be nice to revert it back to 1500
